src/Directory.Build.props

```diff
@@ -45,8 +45,8 @@
     <NETMauiTarget Condition="'$(NETMauiTarget)'==''">net8.0</NETMauiTarget>
     <NetMauiWindowsTargetFramework Condition="'$(NetMauiWindowsTargetFramework)'==''">$(NETMauiTarget)-windows10.0.$(WindowsSDKTargetBuild).0</NetMauiWindowsTargetFramework>
     <NetAndroidTargetFramework Condition="'$(NetAndroidTargetFramework)'==''">$(NETMauiTarget)-android34.0</NetAndroidTargetFramework>
-    <NetCatalystTargetFramework Condition="'$(NetCatalystTargetFramework)'==''">$(NETMauiTarget)-maccatalyst17.0</NetCatalystTargetFramework>
-    <NetiOSTargetFramework Condition="'$(NetiOSTargetFramework)'==''">$(NETMauiTarget)-ios17.0</NetiOSTargetFramework>  
+    <NetCatalystTargetFramework Condition="'$(NetCatalystTargetFramework)'==''">$(NETMauiTarget)-maccatalyst18.0</NetCatalystTargetFramework>
+    <NetiOSTargetFramework Condition="'$(NetiOSTargetFramework)'==''">$(NETMauiTarget)-ios18.0</NetiOSTargetFramework>  
```

Comment:
Should this PR go into v.next branch instead?  It's raising our dependencies beyond what was required for 200.6.0

Reply:
yes we can change that

Reply:
it's actually not a dependency change... if you compile for net8.0, it defaults to 18.0 now. 
![image](https://github.com/user-attachments/assets/0a5fda6f-0fb1-4641-8aab-0e134f2eb34c)